src/services/orgUploadToken/useRegenerateOrgUploadToken.ts

```diff
@@ -46,7 +63,16 @@ export function useRegenerateOrgUploadToken(
         mutationPath: 'regenerateOrgUploadToken',
       })
 
-      return data
+      const parsedRes = RequestSchema.safeParse(data.data)
+      if (!parsedRes.success) {
+        return Promise.reject({
+          status: 404,
+          data: {},
+          dev: 'useRegenerateOrgUploadToken - 404 schema parsing failed',
+        } satisfies NetworkErrorObject)
+      }
+
+      return parsedRes
```

Comment:
iirc, you'll need to return the `.data` field from `parsedRes` after safe parsing it 🤔

Reply:
yeah there was an onSuccess that was further manipulating this

I'm going to put the `useOwner` and org upload token related stuff in new prs for this spun-off ticket - https://github.com/codecov/engineering-team/issues/2857